src/setupDevice.js

```diff
@@ -218,6 +228,16 @@ async function prepareInDFUBootloader(device, dfu) {
 
         try {
             await waitForDevice(device.serialNumber);
+
+            /*
+             * HACK!!!
+             *
+             * Workaround for node-serialport race condition when opening
+             * a USB CDC ACM right after it's been enumerated.
+             *
+             * See https://github.com/node-serialport/node-serialport/issues/1565
+             */
+            await sleep(100);
```

Comment:
The 100 ms sleep is apparently not enough on my PC. It fails with "No such file or directory, cannot open /dev/ttyACM2" when trying to DFU the application.

Upping to 200 ms works in my case. However, maybe we should even increase it a bit more to reduce the risk?

Also, I see there are some linting errors. :wink:

Reply:
OK, let me put that number up to 250msec (and run eslint)